Registry/RepositoryRegistry.php

```diff
@@ -46,6 +46,27 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function names()
+    {
+        return array_keys($this->serviceMap);
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
inherit from where? are you going to add `all` and `names` to RepositoryRegistryInterface?